tools/pipelines/build-client.yml

```diff
@@ -188,7 +188,8 @@ extends:
     interdependencyRange: ${{ parameters.interdependencyRange }}
     packageManagerInstallCommand: 'pnpm i --frozen-lockfile'
     packageManager: pnpm
-    testCoverage: ${{ variables.testCoverage }}
+    testCoverage: true
```

Comment:
Tangent to this PR, no need to take action now: @scarlettjlee @agarwal-navin this is a point to investigate when optimizing the internal pipeline runtime. We want the tests to run with code coverage in the public project so we can calculate the coverage deltas for PRs, but if we don't leverage that data in the internal project we could update this condition to `testCoverage: ${{ eq(variables['System.TeamProject'], 'public' ) }}` if there are significant runtime wins from skipping it in internal.